modules/gql/schema.graphql

```diff
@@ -264,6 +264,7 @@ input LedgerTxFilter {
   byToFrom: String
   byTxId: String
   byTypes: [String!]
+  byAsset: String
```

Comment:
Asset type could probably be a GraphQL enum instead of just a string

Reply:
Resolved in https://github.com/vsc-eco/go-vsc-node/pull/22/commits/9c54403d69490c181d60a67bf4f995983d67bc17